packages/core/src/components/VirtualizedGrid/__stories__/VirtualizedGrid.stories.helpers.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import { Text } from "src/components/Text";
```

Comment:
just noticed this is being imported wrongly
did you ran Storybook to make sure it works?
the import should look something like
```
import { Text } from "../../Text";
```
as we do not have aliases yet in the project

Reply:
My bad.
Just fixed the import statement. Also ran storybook and tests to make sure everything is alright.